Cake.ArgumentHelpers/Cake.ArgumentHelpers.csproj

```
@@ -13,9 +13,13 @@
     <!-- Import shared functionality -->
 	<Import Project="..\Shared.msbuild" />
 	<ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.28.0" PrivateAssets="All" />
-    <PackageReference Include="Cake.Common" Version="0.28.0" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="2.2.0" PrivateAssets="All" />
```

```suggestion
    <PackageReference Include="Cake.Core" Version="2.0.0" PrivateAssets="All" />
```

(same as above) Does this addin need v2.2.0 specifically to work? If not, we should reference v2.0.0 to maximize compatibility with runners as per [best practices](https://cakebuild.net/docs/extending/addins/best-practices#cake-reference)